common/src/main/kotlin/org/jetbrains/kotlinconf/data/Session.kt

```diff
@@ -2,17 +2,18 @@ package org.jetbrains.kotlinconf.data
 
 import kotlinx.serialization.*
 
+// This format is enforced by Sessionize and it should not be changed unless we extract Sessionize DTO
 @Serializable
 data class Session(
-    val id: String,
-    val isServiceSession: Boolean,
-    val isPlenumSession: Boolean,
-    val questionAnswers: List<QuestionAnswer>,
-    val speakers: List<String>,
-    val descriptionText: String?,
-    val startsAt: String,
-    val title: String,
-    val endsAt: String,
-    val categoryItems: List<Int>,
-    val roomId: Int
+        val id: String,
+        val isServiceSession: Boolean,
+        val isPlenumSession: Boolean,
+        val questionAnswers: List<QuestionAnswer>,
+        val speakers: List<String>,
+        val description: String?,
```

Comment:
The description field is not supported on the iOS platform. Could you revert that change?

Reply:
Cannot, just how I commented above - this field is used to read data from Sessionize API so we cannot change it unless we decouple this two data models. Though it works fine.

Reply:
As I wrote in the comment - this instance cannot be changed because it is used to parse Sessionize API. We would need to decouple those two. Though, looks like it works fine in iOS.